tests/cli.rs

```diff
@@ -273,12 +273,21 @@ fn list_mutants_well_tested() {
 }
 
 #[test]
-fn list_mutants_well_tested_name_filter() {
+fn list_mutants_well_tested_examine_name_filter() {
     run()
         .arg("mutants")
         .args(["--list", "--file", "nested_function.rs"])
         .current_dir("testdata/tree/well_tested")
-        .assert_insta("list_mutants_well_tested_name_filter");
+        .assert_insta("list_mutants_well_tested_examine_name_filter");
+}
+
+#[test]
+fn list_mutants_well_tested_exclude_name_filter() {
```

Comment:
Probably we should also test, for at least one type of filter, how slashes are handled. I think `well_tested` has some submodules that you could use.

Reply:
Just to be sure: you mean testing for example excluding a folder like `cargo mutants --exclude /folder/*`, right?

Or what do you mean by `at least one type of filter`?

Reply:
> Just to be sure: you mean testing for example excluding a folder like cargo mutants --exclude /folder/*, right?

Right.

> Or what do you mean by at least one type of filter?

Either includes or excludes.

Like e.g. include one submodule, but then exclude one file within it.

Reply:
I see this is done, thanks!